newIDE/app/src/MainFrame/index.js

```diff
@@ -1064,15 +1130,27 @@ class MainFrame extends React.Component<Props, State> {
       .catch(() => {});
   };
 
-  save = () => {
+  save = (autosave: boolean = false) => {
     saveUiSettings(this.state.editorTabs);
-    if (!this.state.currentProject) return;
+
+    const { currentProject } = this.state;
+    if (!currentProject) return;
     const { i18n } = this.props;
 
     if (this.props.saveDialog) {
       this._openSaveDialog();
+    } else if (this.props.onAutoSaveProject && autosave) {
+      this.props.onAutoSaveProject(
+        currentProject,
+        currentProject.getProjectFile() + '.autosave',
+        err => {
+          this._showSnackMessage(
+            err ? i18n._(t`Autosave Failed!`) : i18n._(t`Autosaved!`)
```

Comment:
Let's not display any message in case of success - otherwise we're flooding the user with message conveying low signal and high noise.
At the end, users are becoming "blind" to the messages and not reading then anymore - missing the important information.

Reply:
aye! :)